app/helpers/assignment_helper.rb

```diff
@@ -99,4 +99,103 @@ def get_team_name_color_in_list_submission(team)
     end
   end
 
+  # Compute total score for this assignment by summing the scores given on all questionnaires.
+  # Only scores passed in are included in this sum.
+  def compute_total_score(scores)
+    total = 0
+    self.questionnaires.each {|questionnaire| total += questionnaire.get_weighted_score(self, scores) }
+    total
+  end
+
+  def compute_reviews_hash
+    @review_scores = {}
```

Comment:
why are these being passed as instance variables, can you refactor all of this into using parameters